spec/naver/searchad/api/ad/service_spec.rb

```diff
@@ -0,0 +1,823 @@
+require 'spec_helper'
+
+include Naver::Searchad::Api
+
+describe Ad::Service do
+  subject(:this) { described_class.new }
+  before(:each) do
+    this.authorization = Auth.get_application_default
+  end
+
+  describe '#list' do
+    context 'when requesting some' do
+      let(:ad_ids) { ['nad-a001-01-000000015970482', 'nad-a001-01-000000015970494'] }
+      before(:each) do
+        stub_request(:get, 'https://api.naver.com/ncc/ads?ids=nad-a001-01-000000015970482,nad-a001-01-000000015970494').
+          to_return(
+            status: 200,
+            headers: {'Content-Type' => 'application/json;charset=UTF-8'},
+            body: <<-JSON
+[
```

Comment:
What about putting all these json in another file that we can load? It's quite hard to read at the moment 😵